doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -1030,7 +1050,17 @@ Additions specific to |NCS|
 zcbor
 =====
 
-|no_changes_yet_note|
+zcbor has been updated from 0.7.0 to 0.8.1.
+Full release notes can be found at:
+https://github.com/zephyrproject-rtos/zcbor/blob/0.8.0/RELEASE_NOTES.md and
+https://github.com/zephyrproject-rtos/zcbor/blob/0.8.1/RELEASE_NOTES.md
```

Comment:
Please add the entries : 
`zcbor 0.8.0 release notes`: https://github.com/zephyrproject-rtos/zcbor/blob/0.8.0/RELEASE_NOTES.md 
`zcbor 0.8.0 release notes`: https://github.com/zephyrproject-rtos/zcbor/blob/0.8.1/RELEASE_NOTES.md 
to doc/nrf/links.txt for the links to work.
```suggestion
For more information see the `zcbor 0.8.0 release notes`_ and the `zcbor 0.8.1 release notes`_.
```